tests/integration/golang/aim/run/get_run_info_test.go

```diff
@@ -26,20 +28,53 @@ func TestGetRunInfoTestSuite(t *testing.T) {
 
 func (s *GetRunInfoTestSuite) SetupTest() {
 	s.BaseTestSuite.SetupTest()
-
-	var err error
-	s.run, err = s.RunFixtures.CreateExampleRun(context.Background(), s.DefaultExperiment)
-	s.Require().Nil(err)
 }
 
 func (s *GetRunInfoTestSuite) Test_Ok() {
+	// create test data.
+	experiment, err := s.ExperimentFixtures.CreateExperiment(context.Background(), &models.Experiment{
+		Name:           uuid.New().String(),
+		NamespaceID:    s.DefaultNamespace.ID,
+		LifecycleStage: models.LifecycleStageActive,
+	})
+	s.Require().Nil(err)
+
+	run, err := s.RunFixtures.CreateRun(context.Background(), &models.Run{
+		ID:             "id",
+		Name:           "TestRun",
+		Status:         models.StatusScheduled,
+		StartTime:      sql.NullInt64{Int64: 123456789, Valid: true},
+		EndTime:        sql.NullInt64{Int64: 123456789, Valid: true},
+		SourceType:     "JOB",
+		ExperimentID:   *experiment.ID,
+		LifecycleStage: models.LifecycleStageActive,
+	})
+	s.Require().Nil(err)
+
+	metricContext, err := s.ContextFixtures.CreateContext(context.Background(), &models.Context{
+		Json: datatypes.JSON(`{"key": "key", "value": "value"}`),
+	})
+	s.Require().Nil(err)
+
+	_, err = s.MetricFixtures.CreateLatestMetric(context.Background(), &models.LatestMetric{
+		Key:       "key",
+		Value:     123,
+		Timestamp: 123456789,
+		Step:      1,
+		IsNan:     false,
+		RunID:     run.ID,
+		ContextID: common.GetPointer(metricContext.ID),
```

Comment:
this association will be automatic with Gorm, if the above `metricContext` is just created here as `Context`

Reply:
well -- maybe -- verifying the same thing in my PR